_posts/2020-08-07-g2p.md

```diff
@@ -392,40 +406,215 @@ You can also use abbreviations like this in the G2P studio by writing them in th
 
 ### Regular Expressions
 
-[Regular expressions](https://en.wikipedia.org/wiki/Regular_expression) are used ubiquitously in programming to do define certain search patterns in text. In fact, this is how `g2p` rules work! They eventually get compiled into a regular expression too. For the most part, you can add basic regular expressions to your rules
+[Regular expressions](https://en.wikipedia.org/wiki/Regular_expression) are used ubiquitously in programming to define certain search patterns in text. In fact, this is how `g2p` rules work! They eventually get compiled into a regular expression too. For the most part, you can add regular expression syntax to your rules. So, suppose you wanted to write a rule that deleted word-final 's', you could write the following:
+
+| in | out | context_before | context_after |
+|---|---|---|---|
+| s |  | \S | \s\|$ |
```

Comment:
yes, but we had a problem in readalongs with it, because \b gives false positives for some orthographies, so I'm hesitant about showing examples with it.